src/module/Phpug/public/js/phpug.js

```diff
@@ -125,6 +125,7 @@ var cfp = L.layerJSON({
     },
     onEachMarker : function(e, marker) {
         oms.addMarker(marker);
```

Comment:
Necessary as the individual functions are slightly different but this call has to be added to all four of them :disappointed: